lib/service_manager.dart

```diff
@@ -464,43 +514,51 @@ class ServiceExtensionManager {
   }
 
   StreamController<bool> _getServiceExtensionController(String name) {
-    return _getStream(name, _serviceExtensionController,
-        onFirstListenerSubscribed: () {
-      // If the service extension is in [_serviceExtensions], then we have been
-      // waiting for a listener to add the initial true event. Otherwise, the
-      // service extension is not available, so we should add a false event.
-      _serviceExtensionController[name].add(_serviceExtensions.contains(name));
-    });
+    return _getStream(
```

Comment:
these flutter style tweaks are all an improvement but in the future try to leave formatting changes for separate CLs as reviewers can review formatting only changes much more quickly if they know that is all that is in the CL.

Reply:
Noted, thanks.